vault/resource_ldap_secret_backend_static_role.go

```diff
@@ -24,7 +24,7 @@ func ldapSecretBackendStaticRoleResource() *schema.Resource {
 			Description:  "The path where the LDAP secrets backend is mounted.",
 			ValidateFunc: provider.ValidateNoLeadingTrailingSlashes,
 		},
-		consts.FieldRole: {
+		consts.FieldRoleName: {
```

Comment:
Changing this to `role_name` to align with the docs and with dynamic role resource

Reply:
Is changing the name of this going to cause all existing static roles to be destroyed and recreated?

Reply:
This is a good question. I _think_ the only concern would be when importing an ldap resource into the tfvp. In that case I think we are okay since the API field that we get from Vault is expected to be `role_name`. But we should make sure to test this

Reply:
That's true. I forgot that the static account support wasn't live already. I don't think this is an issue then 👍